auth/browser.go

```diff
@@ -63,11 +63,11 @@ func NewBrowserAuth(ctx context.Context, opts ...Option) (BrowserAuth, error) {
 	} else {
 		br.opts.workspace = wsp
 	}
-	auther, err := browser.New(br.opts.workspace, browser.OptBrowser(br.opts.browser), browser.OptTimeout(br.opts.loginTimeout), browser.OptVerbose(br.opts.verbose))
+	author, err := browser.New(br.opts.workspace, browser.OptBrowser(br.opts.browser), browser.OptTimeout(br.opts.loginTimeout), browser.OptVerbose(br.opts.verbose))
```

Comment:
Can this be added to ignores too please?

Reply:
oh, I should have read more of context here to understand what was wanted.
I will add it to skip.

Reply:
rebased, redone, force-pushed